fragment_test.go

```diff
@@ -256,6 +256,79 @@ func TestFragment_FieldSum(t *testing.T) {
 	})
 }
 
+// Ensure a fragment can find the max of field values.
+func TestFragment_FieldMinMax(t *testing.T) {
+	const bitDepth = 16
+
+	f := test.MustOpenFragment("i", "f", pilosa.ViewStandard, 0, "")
+	defer f.Close()
+
+	// Set values.
+	if _, err := f.SetFieldValue(1000, bitDepth, 382); err != nil {
+		t.Fatal(err)
```

Comment:
I don't expect problems now, but maybe these values should include some negative numbers.

Reply:
@alanbernstein there are no negative values at the fragment level; that offset it applied by the frame. I added a negative number to the test in `executor.go`: 47a5ed84bd70e78d480b6d308ba2589c044b3c81